pkg/gatewayserver/gatewayserver_test.go

```diff
@@ -113,9 +116,7 @@ func TestGatewayServer(t *testing.T) {
 			WSPingInterval: wsPingInterval,
 		},
 		UpdateConnectionStatsDebounceTime: 0,
-		Stats: &gsredis.GatewayConnectionStatsRegistry{
-			Redis: statsRedisClient,
-		},
+		Stats:                             statsRegistry,
```

Comment:
Out of curiosity; why this diff?

Reply:
In a previous iteration `statsRegistry` was used below for some debugging. Was left because it looks like the code is a bit more clear.